### ISSUE TITLE
`MemEff` = 0% if I add argument `-s TIMEOUT`

### ISSUE THREAD

**Issue by aowenson-imm** [troycomi/reportseff#58]
Version 2.7.6

> reportseff -u $U

```
    JobID    State       Elapsed  TimeEff   CPUEff   MemEff 
  xxxxxxx   TIMEOUT     17:00:06  100.0%     5.0%    72.2%  
  xxxxxxx   TIMEOUT     15:00:18  100.0%    16.0%    74.5%  
...
```

Good. But if I run:

> reportseff -u $U -s TIMEOUT

Then all the `MemEff` values are zeroed, at least for `TIMEOUT` jobs:

```
    JobID   State      Elapsed  TimeEff   CPUEff   MemEff 
  xxxxxxx  TIMEOUT    17:00:06  100.0%     5.0%     0.0%  
  xxxxxxx  TIMEOUT    15:00:18  100.0%    16.0%     0.0%  
...
  ```

**Comment by troycomi** [comment]
O weird! Can you provide the `--debug` output in both cases for the job of interest?

**Comment by aowenson-imm** [comment]
debug output is identical for both cases, but is the structure the clue?
```
^  ^2^  ^01:00:29^  ^xxxxxxx^        ^xxxxxxx^        ^^           ^1^  ^^   ^40G^  ^TIMEOUT^    ^01:00:00^  ^01:01:06
^  ^2^  ^01:00:33^  ^xxxxxxx.batch^  ^xxxxxxx.batch^  ^19855760K^  ^1^  ^1^  ^^     ^CANCELLED^  ^^          ^01:01:06
```

All our batch jobs appear like this in sacct output - split over 2 rows. I think this is due to our use of prolog & epilog.

**Comment by troycomi** [comment]
Great, I should be able to troubleshoot from here.  Discarding the second line is my suspicion, but the job status *should* be controlled by the top line.

I have a larger change I want to push through first, but this will be the next fix.

**Comment by troycomi** [comment]
Should be fixed with v2.7.7.  Please reach out if you have any other issues, thanks for submitting this bug!

**Comment by aowenson-imm** [comment]
Confirmed fixed.
